app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserDialog.java

```diff
@@ -299,4 +318,30 @@ public void onStart() {
             w.setLayout(WindowManager.LayoutParams.MATCH_PARENT, WindowManager.LayoutParams.MATCH_PARENT);
         }
     }
+
+    private static void setSubtitleApprearance(final Toolbar toolbar) {
```

Comment:
Find and set a dialog subtitle appearance. This is a little hacky but it works correctly every time.